### ISSUE TITLE
One of the three cards is not wiped. There's an error in the logs

### ISSUE THREAD

**Issue by Perlover** [NicolasDorier/boltcards-plugin#13]
Boltcards Extension - v1.1.5.0 

I created three Bolt Cards. One of the cards did not have all green checkmarks, and one of the penultimate checkmark in the list was red.

Then I tried to pay with one of the cards - the first payment went through. On the other cards the payments were hanging.

Then I decided to wipe the cards. The first and the third were successfully wiped. The second card, from which I successfully paid, did not want to be wiped - the application "Bolt Card NFC Card Creator" was hanging.

I looked in the BTCPay logs (`docker logs -f generated_btcpayserver_1`) - there was an error:

```
fail: Microsoft.AspNetCore.Server.Kestrel: Connection id "0HN510EC0TLUR", Request id "0HN510EC0TLUR:00000001": An unhandled exception was thrown by the application.
System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values. (Parameter 'version')
   at BTCPayServer.NTag424.IssuerKey.CreateCardKey(Byte[] uid, Int32 version, Byte[] additionalData)
   at BTCPayServer.Extensions.CreatePullPaymentCardKey(IssuerKey issuerKey, Byte[] uid, Int32 version, String pullPaymentId) in /source/BTCPayServer/Extensions.cs:line 123
   at BTCPayServer.Plugins.BoltcardFactory.Controllers.APIBoltcardFactoryController.RegisterBoltcard(String appId, RegisterBoltcardRequest request, String onExisting) in /build/BTCPayServer.Plugins.Boltcards/BoltcardFactory/Controllers/APIBoltcardFactoryController.cs:line 119
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(ActionContext actionContext, IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextExceptionFilterAsync>g__Awaited|26_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ExceptionContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|25_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at BTCPayServer.Hosting.BTCPayMiddleware.Invoke(HttpContext httpContext) in /source/BTCPayServer/Hosting/BTCpayMiddleware.cs:line 97
   at BTCPayServer.Hosting.GreenfieldMiddleware.Invoke(HttpContext httpContext) in /source/BTCPayServer/Hosting/GreenfieldMiddleware.cs:line 30
   at Microsoft.AspNetCore.Diagnostics.StatusCodePagesMiddleware.Invoke(HttpContext context)
   at BTCPayServer.Hosting.HeadersOverrideMiddleware.Invoke(HttpContext httpContext) in /source/BTCPayServer/Hosting/HeadersOverrideMiddleware.cs:line 29
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.ProcessRequests[TContext](IHttpApplication`1 application)
```

**Comment by Perlover** [comment]
Small clarification: whether it matters or not, but when I programmed the cards, this was the order:

1) The first card was programmed, but the penultimate check mark was red. I paid attention to this
2) I held up the second blank card - all the ticks were green
3) I decided to bring the first card again - reprogramming failed - as I understood then from the messages that it was programmed after all.
4) I brought the third card for programming and it was successfully programmed again.

The point of my clarification is that maybe because of points 1 and 3 I "knocked" the counter inside in BTCPay plugin? I don't know what the counting card was when I programmed it, which I can't clear now. Maybe it was the third, or maybe it was the first one during programming...
